### DISCUSSION TITLE
BE ip change【BE ip变更，如何进行数据迁移】

### DISCUSSION THREAD

**Discussion by big-doudou** [apache/doris#33452] [A - General / Q&A]
The cluster has 3 replicas, can I just STOP a BE node, Drop it, wait for the ip change to complete and then rejoin it.
Is it necessary to delete the data on the BE node?
Will FE incrementally sync the data during the downtime when doing tablet balanced scheduling, or is it considered to be a new BE node after the ip change and need to clean up the data on the BE node and then sync it in full?

集群有3副本，可以直接stop一个BE节点，Drop后，等待ip变更完成后，再重新加入吗。
是否需要删除BE节点上的数据？
FE在进行tablet 均衡调度的时候，会增量同步停机期间的数据，还是认为ip变更后是新的BE节点，需要清理BE节点上数据后，全量同步？

**Comment by yagagagaga** [comment]
> 集群有3副本，可以直接stop一个BE节点，Drop后，等待ip变更完成后，再重新加入吗。

可以，ip变更后，就认为是一台新的节点了

**Reply by big-doudou** [reply]
是否需要删除BE节点上的数据？
FE在进行tablet 均衡调度的时候，会增量同步停机期间的数据，还是认为ip变更后是新的BE节点，需要清理BE节点上数据后，全量同步？

**Reply by yagagagaga** [reply]
直接把那台be节点drop掉，停止be进程，变更IP后，重新启动一个全新的be进程，加入集群就可以了
